server/game/gamesteps/challenge/challengeflow.js

```diff
@@ -17,6 +17,7 @@ class ChallengeFlow extends BaseStep {
             new SimpleStep(this.game, () => this.announceChallenge()),
             new SimpleStep(this.game, () => this.promptForAttackers()),
             new SimpleStep(this.game, () => this.announceAttackerStrength()),
+            new SimpleStep(this.game, () => this.chooseStealthTargets()),
```

Comment:
Timing on this is now incorrect though - per the rules, the declaration of stealth targets is done at the time of choosing attackers but prior to the challenge officially being initiated. Hence, if I declare a challenge using Core Tyrion, the sequence should be:
```
Pick 'Intrigue' challenge
Choose Tyrion
Choose stealth target for Tyrion
Challenge is declared
Tyrion gains 2 gold
```

Moving it out here will inappropriately change the flow to:
```
Pick 'Intrigue' challenge
Choose Tyrion
Challenge is declared
Tyrion gains 2 gold
Choose stealth target for Tyrion
```